ts/features/pn/components/MessageF24.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { View } from "react-native";
+import { Body, ButtonLink, VSpacer } from "@pagopa/io-app-design-system";
+import { constNull } from "fp-ts/lib/function";
+import I18n from "../../../i18n";
+import { UIAttachment } from "../../../store/reducers/entities/messages/types";
+import { MessageAttachments } from "../../messages/components/MessageAttachments";
+import { PnMessageDetailsSection } from "./PnMessageDetailsSection";
+
+type Props = {
+  attachments: ReadonlyArray<UIAttachment>;
+};
+
+export const MessageF24 = ({ attachments }: Props) => {
+  const renderContent = () => {
```

Comment:
What do you think of either:
- not using a function for this but to move it directly inside the main rendering `return` statement, or
- moving this function outside of the main rendering function, as to avoid defining a new function every time the main rendering function is invoked, or
- caching this function with a `useCallback` hook
?

Reply:
Changed https://github.com/pagopa/io-app/pull/5156/commits/91b6fdd3392ca585ac026ae10e0d8256d8357943